modules/massrole/commands/massrole.js

```diff
@@ -0,0 +1,202 @@
+                    member.roles.remove(interaction.options.getRole('role'));
+                }
+            });
+            await interaction.editReply(localize('massrole', 'done'));
+        }
+        if (target === 'humans') {
+            await interaction.deferReply({ ephemeral: true });
+            interaction.guild.members.cache.forEach(member => {
+                if (member.manageable) {
+                    if (!member.user.bot) {
+                        member.roles.remove(interaction.options.getRole('role'));
+                    }
+                }
+            });
+            await interaction.editReply(localize('massrole', 'done'));
+        }
+    },
+    'remove-all': async function (interaction) {
+        await checkTarget(interaction);
+        if (target === 'all') {
+            await interaction.deferReply({ ephemeral: true });
+            interaction.guild.members.cache.forEach(member => {
+                member.roles.remove(member.roles.cache.filter(role => !role.managed));
+            });
+            await interaction.editReply(localize('massrole', 'done'));
+        } else if (target === 'bots') {
+            await interaction.deferReply({ ephemeral: true });
+            interaction.guild.members.cache.forEach(member => {
+                if (member.manageable) {
+                    if (member.user.bot) {
+                        member.roles.remove(member.roles.cache.filter(role => !role.managed));
+                    }
+                }
+            });
+            await interaction.editReply(localize('massrole', 'done'));
+        } else if (target === 'humans') {
+            await interaction.deferReply({ ephemeral: true });
+            interaction.guild.members.cache.forEach(member => {
+                if (member.manageable) {
+                    if (!member.user.bot) {
+                        member.roles.remove(member.roles.cache.filter(role => !role.managed));
+                    }
+                }
+            });
+            await interaction.editReply(localize('massrole', 'done'));
+        }
+    }
+};
+
+/**
+ * Read content of "target"-option
+ *
+ */
+function checkTarget(interaction) {
+    if (!interaction.options.getString('target') || interaction.options.getString('target') === 'all') {
+        target = 'all';
+    } else if (interaction.options.getString('target') === 'bots') {
+        target = 'bots';
+    } else if (interaction.options.getString('target') === 'humans') {
+        target = 'humans';
+    }
+}
+
+/**
+ * Make autocomplete for "target"-option in commands
+ *
+ */
+function autoCompleteTarget(interaction) {
+    interaction.respond([
+        {
+            name: localize('massrole', 'all-users'),
+            value: 'all'
+        },
+        {
+            name: localize('massrole', 'bots'),
+            value: 'bots'
+        },
+        {
+            name: localize('massrole', 'humans'),
+            value: 'humans'
+        }
+    ]);
+}
```

Comment:
As this auto-complete does not query personalized data, please use [choises](https://discord.js.org/#/docs/discord.js/stable/typedef/ApplicationCommandOptionChoice) on the command instead. This will increase performance and UX.